app/client/lib/highlight.js

```diff
@@ -1,5 +1,4 @@
 var find = require('findandreplacedomtext')
-var css = require('dom-css')
 
 // Highlights all the words in text inside a node
 function Highlight (node, text, wrapper) {
```

Comment:
**also** (thoughts trickling in slowly here 😄), `text` can be `* blah blah`, when the user is searching just the local collection. So:

```js
text = text.split(' ')
```

should be:

```js
text = text.replace('*', '').trim().split(' ')
```